style.css

```diff
@@ -16,25 +17,23 @@ html {
 }
 
 .nav-bar::after {
-  position: absolute;
-  bottom: -50px;
-  left: 0px;
-  width: 100%;
-  content:" ";
-  background:
-  radial-gradient(circle at 50% 0%, #BD4B4B 25%, #EEEEEE 26%, #EEEEEE 40%);
-  background-color: #EEEEEE ;
-  background-size: 30px 100px;
-  height: 50px;
-  background-repeat: repeat-x;
+ 	 position: absolute;
+	  bottom: -50px;
```

Comment:
this has an extra space at the start? indentation is different than the rest for some reason